src/main/java/org/broadinstitute/hellbender/utils/clipping/ClippingOp.java

```diff
@@ -39,6 +39,11 @@ public int getLength() {
      * @param originalRead the read to be clipped
      */
     public GATKRead apply(final ClippingRepresentation algorithm, final GATKRead originalRead) {
+        return apply(algorithm, originalRead, true);
+    }
+
+    // Warning, this method is intended for testing purposes
+    GATKRead apply(final ClippingRepresentation algorithm, final GATKRead originalRead, final boolean runAsserts) {
```

Comment:
Since it looks like the only test that uses this is currently turned off, could we wait to make all of these runAssert changes until we want to include that test?